spec/integration_spec.rb

```diff
@@ -148,6 +148,15 @@
             expect(subject.license).to eql(license)
           end
         end
+
+        context 'BSL' do
```

Comment:
The integration test and corresponding fixture aren't really necessary because this PR really just changes license file matching, which is tested for in `license_file_spec.rb`, but it doesn't hurt either.